be/src/storage/tablet_manager.cpp

```diff
@@ -1018,12 +1018,22 @@ Status TabletManager::start_trash_sweep() {
                 TabletSharedPtr& tablet = *it;
                 std::string tablet_path = tablet->tablet_path();
                 if (Env::Default()->path_exists(tablet_path).ok()) {
-                    // take snapshot of tablet meta
-                    std::string meta_file_path = path_util::join_path_segments(
-                            tablet->tablet_path(), std::to_string(tablet->tablet_id()) + ".hdr");
-                    tablet->tablet_meta()->save(meta_file_path);
-                    OLAPStatus rm_st = move_to_trash(tablet_path, tablet_path);
-                    if (rm_st == OLAP_SUCCESS) {
+                    if (tablet->keys_type() == KeysType::PRIMARY_KEYS) {
+                        Status st = SnapshotManager::instance()->write_meta_snapshot(tablet);
+                        if (!st.ok()) {
+                            LOG(WARNING) << "Fail to snapshot_trash, tablet_id=" << tablet->tablet_id()
+                                         << " schema_hash=" << tablet->schema_hash() << ", status=" << st.to_string();
+                        } else {
+                            LOG(INFO) << "Created snapshot tablet_id=" << tablet->tablet_id()
```

Comment:
This log has little help and you can remove it, otherwise, lots of annoying logs will be generated.

Reply:
done.